### PR TITLE
[IMP]website: Remove Header between parent and child template options an...

### PR DESCRIPTION
**Pull request by vidhin6**
...d childs menu option must be false if parent menu option is set false in customize menu

Task: https://www.odoo.com/web#id=12298&view_type=form&model=project.task&action=333&active_id=131
Pad:
https://pad.odoo.com/p/r.mba2VdtRZCyy7RYS

### CODE REVIEW

File: addons/website/models/ir_ui_view.py [hunk 1]
```diff
@@ -293,6 +295,11 @@ def customize_template_get(self, cr, uid, key, full=False, bundles=False, contex
                 })
         return result
 
+    def toggle_options(self, cr, uid, ids, context=None):
+        view_ids = self.search(cr, uid, [('inherit_id', 'child_of', ids), ('active', '=', True)], context=context)
```
Comment (Gorash) [inline addons/website/models/ir_ui_view.py]: If a user unactive a parent template (like the footer), all inherit are unactivate but if this template haven't customize_show, the user can't re-activate the templates.
Please fix the domain

File: addons/website/models/ir_ui_view.py [hunk 2]
```diff
@@ -293,6 +295,11 @@ def customize_template_get(self, cr, uid, key, full=False, bundles=False, contex
...
+        ids = view_ids if view_ids and len(view_ids) else ids
+        self.toggle(cr, uid, ids, context=context)
```
Comment (Gorash) [inline addons/website/models/ir_ui_view.py]: remove the line 300 and use self.toggle(cr, uid, view_ids or ids, context=context)

Comment (gan-odoo) [inline addons/website/models/ir_ui_view.py]: OK Done!

File: addons/website/models/ir_ui_view.py (hunk 1 above)
Comment (gan-odoo) [inline addons/website/models/ir_ui_view.py]: Yes, I agree with you.
Done!

**Comment by Gorash** [comment]
@tde-banana-odoo branch to merge

**Pull request closed without merging**